v2/plugins/docusaurus-plugin-content-pages/__tests__/index.test.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import path from 'path';
+
+import loadSetup from '@test/loadSetup';
+import DocusaurusPluginContentPages from '../index';
+
+describe('docusaurus-plugin-content-pages', () => {
+  describe('loadContents', () => {
+    test.each([
```

Comment:
I'm trying out `test.each` here to reduce repetition. Lemme know if you don't like it. I don't have strong objections against going back to the former approach.